.github/workflows/build-neovide.yml

```diff
@@ -0,0 +1,17 @@
+name: build flatpak
+on: [push, pull_request]
+jobs:
+  build:
+    name: build flatpak
+    runs-on: ubuntu-latest
```

Comment:
why?

Reply:
> why?

1. make sure everything successfully building outside of my computer

2. it's very convenient to see a green check mark when everything is ok, and a red cross when everything is broken

it's not necessary, i can remove if you want

Reply:
the buildbot is going to do that for you, remove this.

Reply:
removed https://github.com/flathub/flathub/pull/5745/commits/e3e915db51b27f1f48c791b5edca2830490b8cbe